.github/workflows/create-release-official.yml

```diff
@@ -1,59 +1,59 @@
-name: Create new release
-
-on:
-  # run when a release is published,
-  # which creates a new tag
-  push:
-    tags:
-      - v2*
-
-jobs:
-  build-and-push:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0 # required to access tags
-          submodules: "true"
-
-      - name: Build & run devcontainer image
-        # this always builds a new image from scratch rather than from the build-devcontainer-image workflow output
-        # so that we pick up the latest versions of everything
-        # NB: if you update this also update live-validation.yml
-        id: devcontainer
-        run: |
-          docker build --tag devcontainer:latest .devcontainer
-          mkdir -p $HOME/.docker # in case it doesn't exist
-          container_id=$(docker create -w /workspace -v $GITHUB_WORKSPACE:/workspace -v /var/run/docker.sock:/var/run/docker.sock devcontainer:latest)
-          docker start "$container_id"
-          echo "container_id=$container_id" >> $GITHUB_ENV
-
-      - name: Build required release files
-        run: |
-          container_id=${{env.container_id}}
-          docker exec "$container_id" task make-release-artifacts
-
-      - name: Upload release assets
-        uses: svenstaro/upload-release-action@7319e4733ec7a184d739a6f412c40ffc339b69c7 # this is v2.5.0, but pinned
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          tag: ${{ github.ref }}
-          file: "v2/out/release/*"
-          file_glob: true
-
-      - name: Docker login
-        run: |
-          container_id=${{env.container_id}}
-          docker exec -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e DOCKER_REGISTRY "$container_id" task docker-login
-        env:
-          DOCKER_REGISTRY: ${{ secrets.REGISTRY_LOGIN }}
-          AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-          AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
-
-      - name: Build, tag and push docker image
-        run: |
-          container_id=${{env.container_id}}
-          docker exec -e DOCKER_PUSH_TARGET "$container_id" task controller:docker-push-multiarch
-        env:
-          DOCKER_PUSH_TARGET: ${{ secrets.REGISTRY_PUBLIC }}
```

Comment:
Weird how GitHub is showing this diff -- from what I can tell you just changed the comments is that right?

Reply:
Correct, and I renamed the file